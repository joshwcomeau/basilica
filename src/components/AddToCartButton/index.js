// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';

import Button from '../Button';
import './index.scss';


const AddToCartButton = ({ price }) => {
  return (
    <Button
      icon="add_to_cart"
      theme="dark"
      size="medium"
      withInnerBorder
    >
      {price} | Add To Cart
    </Button>
  );
};

AddToCartButton.propTypes = {
  price: PropTypes.string,
};

AddToCartButton.defaultProps = {

};

export default AddToCartButton;
