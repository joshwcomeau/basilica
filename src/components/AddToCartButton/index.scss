@import '../../variables';

.add-to-cart-button {


  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
