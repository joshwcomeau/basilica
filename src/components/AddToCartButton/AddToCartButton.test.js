/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import AddToCartButton from './index';

describe('AddToCartButton', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AddToCartButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
