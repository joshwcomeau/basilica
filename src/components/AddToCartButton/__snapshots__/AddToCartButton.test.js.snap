exports[`AddToCartButton renders correctly 1`] = `
<div
  className="add-to-cart-button">
  Your Component Here :)
</div>
`;
